### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by AlessandroTischer** [AlessandroTischer/tuya_climate_mult_edit#4]
Ho @AlessandroTischer, you will continue to keep everything updated in parallel to the official branch? Grazie

_Originally posted by @dany-log4u in https://github.com/AlessandroTischer/tuya_climate_mult_edit/issues/3#issuecomment-2469137780_
            

Did they make any significant update? I'm not checking the core integration frequently as I only use my version. If I find the time I might update it in the next weeks...

BTW, please create a new issue for things not related, just to keep things organized ;-)

**Comment by AlessandroTischer** [comment]
Updated to source.
